simapp/sim_test.go

```diff
@@ -48,11 +54,13 @@ func init() {
 // helper function for populating input for SimulateFromSeed
 func getSimulateFromSeedInput(tb testing.TB, w io.Writer, app *SimApp) (
```

Comment:
sighh...this function is becoming overwhelmingly verbose and hard to grok.

Reply:
agree... I'd like to do some refactors later on the simulator and this is one of the things that'd be cleaned up